### ISSUE TITLE
[p-fileUpload] Always delete the first file when user try to do upload files

### ISSUE THREAD

**Issue by bluegray1015** [primefaces/primeng#2134]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
After choosing the different files for file uploading, user want to delete one of the files in `[p-fileUpload]` panel. When user click the delete button of any file, it always delete the first file.

**Expected behavior**
When user click the delete button of the second file, the second file should be deleted. When user click the delete button of the last file, the last file should be deleted.

**Minimal reproduction of the problem with instructions**
1.Choose the more than 1 file for file uploading in `[p-fileUpload]`.
2.Delete one of the files excepts the first one in file upload panel.
3.No matter what file user delete, it always delete the first one.

* **Angular version:** 2.0.X
2.4.6

* **PrimeNG version:** 2.0.X
2.0.1

* **Browser:**
Google Chrome 56.0.2924.87 (64-bit)

**Comment by pauly815** [comment]
This looks to be resolved in PrimeNG version 2.0.3.

**Comment by cagataycivici** [comment]
This should be fixed in recent PrimeNG release, please try with it and if the issue persists drop a comment with along with a test case based on plunkr below and we'll review again.

http://plnkr.co/edit/Wj39h1?p=preview
